### ISSUE TITLE
Loading ED25519 key from file throws KeyError

### ISSUE THREAD

**Issue by alexfrostdesu** [latchset/jwcrypto#182]
Hi, I want to use EdDSA for my JWT and getting an error after trying to load key from file.

Here is the error:
```
File "/Users/matveyloshakov/project/apiuser/views.py", line 300, in post
    key = jwk.JWK.from_pem(SECRET_KEY)
  File "/Users/matveyloshakov/project/backend.env/lib/python3.6/site-packages/jwcrypto/jwk.py", line 904, in from_pem
    obj.import_from_pem(data, password)
  File "/Users/matveyloshakov/project/backend.env/lib/python3.6/site-packages/jwcrypto/jwk.py", line 851, in import_from_pem
    self.import_from_pyca(key)
  File "/Users/matveyloshakov/project/backend.env/lib/python3.6/site-packages/jwcrypto/jwk.py", line 818, in import_from_pyca
    self._import_pyca_pri_okp(key)
  File "/Users/matveyloshakov/project/backend.env/lib/python3.6/site-packages/jwcrypto/jwk.py", line 456, in _import_pyca_pri_okp
    crv=params['crv'],
KeyError: 'crv'
```

Here is how I generate the SECRET_KEY:
```
openssl genpkey -algorithm ED25519 -out secret_key.pem
```

Here is how I load the key:
```
SECRET_KEY = open(os.path.join(script_dir, 'secret_key.pem'), 'rb').read()
key = jwk.JWK.from_pem(SECRET_KEY)
```

What I am doing wrong or what is the reason this does not work? I can perfectly generate and use ECDSA/RSA keys using the same method, but not EdDSA.

**Comment by simo5** [comment]
This was fixed in #178 but we haven't made a release yet

**Comment by simo5** [comment]
Duplicate of #176
